pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[tool.poetry]
+name = "sceptre-file-resolver"
+version = "1.0.6"
+description = "A Sceptre resolver to retrieve file content"
+authors = ["zaro0508 <zaro0508@gmail.com>"]
+readme = "README.md"
+homepage = "https://github.com/Sceptre/sceptre-file-resolver"
+license = "Apache-2.0"
+packages = [{ include = "resolver" }]
+keywords = ["sceptre", "sceptre-resolver"]
+classifiers = [
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "Environment :: Console",
+]
+
+[tool.poetry.dependencies]
+python = ">=3.8,<3.13"
+requests = "^2.25"
+# sceptre = ">=4.0.1"
+sceptre = { git = "https://github.com/branchvincent/Sceptre", rev = "py312" }
```

Comment:
this project should not depend on your branch.

Reply:
Reverted, this was just a temporary change to test 3.12 support. Since the latest `sceptre` doesn't allow 3.12, this package has to wait for a new release with https://github.com/Sceptre/sceptre/pull/1390 due to the conflicting requirements:

```console
$ poetry add sceptre@latest
Using version ^4.3.0 for sceptre

Updating dependencies
Resolving dependencies... (0.0s)

The current project's supported Python range (>=3.8,<3.13) is not compatible with some of the required packages Python requirement:
  - sceptre requires Python >=3.7,<3.12, so it will not be satisfied for Python >=3.12,<3.13
```

Does sceptre need such a tight upper bound? Loosening it to `^3.8` would avoid this